component/service/src/main/java/org/exoplatform/social/rest/impl/user/UserRestResourcesV1.java

```diff
@@ -1498,6 +1498,7 @@ private String importUser(UserImportResultEntity userImportResultEntity,
       if (StringUtils.isNotBlank(groups)) {
         List<String> groupsList = Arrays.asList(groups.split(";"));
         for (String groupMembershipExpression : groupsList) {
+          if (groupMembershipExpression.equals("/platform/externals")) continue;
```

Comment:
```suggestion
          if (groupId.equals("/platform/externals")) continue;
```
This line should be after the groupId definition